databallpy/load_data/tracking_data/tracab.py

```diff
@@ -0,0 +1,264 @@
+
+    file.close()
+
+    size_lines = len(lines)
+
+    data = {
+        "timestamp": [np.nan] * size_lines,
+        "ball_x": [np.nan] * size_lines,
+        "ball_y": [np.nan] * size_lines,
+        "ball_z": [np.nan] * size_lines,
+        "ball_speed": [np.nan] * size_lines,
+        "ball_status": [None] * size_lines,
+        "ball_posession": [None] * size_lines,
+    }
+
+    if verbose:
+        print("Writing lines to dataframe:")
+
+    for idx, line in enumerate(tqdm(lines)):
+
+        timestamp, players_info, ball_info, _ = line.split(":")
+        data["timestamp"][idx] = int(timestamp)
+
+        players = players_info.split(";")[:-1]
+        for player in players:
+            data = _add_player_data_to_dict(player, data, idx)
+
+        data = _add_ball_data_to_dict(ball_info, data, idx)
+
+    df = pd.DataFrame(data)
+
+    for col in df.columns:
+        if "_x" in col or "_y" in col or "_z" in col or "_speed" in col:
+            df[col] /= 100
+            df[col] = np.round(df[col], 3)
+        else:
+            pass
+
+    df = _insert_missing_rows(df)
+
+    return df
+
+
+def _add_player_data_to_dict(player: str, data: dict, idx: int) -> dict:
+    """Function that adds the data of one player to the data dict for one frame
+
+    Args:
+        player (str): containing data from player
+        data (dict): data dictionary to write results to
+        idx (int): indicates position in data dictionary
+
+    Returns:
+        dict: contains all tracking data
+    """
+
+    team_id, _, shirt_num, x, y, speed = player.split(",")
+
+    team_ids = {0: "home", 1: "away"}
+    team = team_ids.get(int(team_id))
+    if team is None:  # player is unknown or referee
+        return data
+
+    if f"{team}_{shirt_num}_x" not in data.keys():  # create keys for new player
+        data[f"{team}_{shirt_num}_x"] = [np.nan] * len(data["timestamp"])
+        data[f"{team}_{shirt_num}_y"] = [np.nan] * len(data["timestamp"])
+        data[f"{team}_{shirt_num}_speed"] = [np.nan] * len(data["timestamp"])
+
+    data[f"{team}_{shirt_num}_x"][idx] = int(x)
+    data[f"{team}_{shirt_num}_y"][idx] = int(y)
+    data[f"{team}_{shirt_num}_speed"][idx] = float(speed)
+
+    return data
+
+
+def _add_ball_data_to_dict(ball_info: str, data: dict, idx: str) -> dict:
+    """Function that adds the data of the ball to the data dict for one frame
+
+    Args:
+        ball_info (str): containing data from the ball
+        data (dict): data dictionary to write results to
+        idx (int): indicates position in data dictionary
+
+    Returns:
+        dict: contains all tracking data
+    """
+
+    x, y, z, speed, posession, status = ball_info.split(";")[0].split(",")[:6]
+    data["ball_x"][idx] = int(x)
+    data["ball_y"][idx] = int(y)
+    data["ball_z"][idx] = int(z)
+    data["ball_speed"][idx] = float(speed)
+    data["ball_posession"][idx] = posession
+    data["ball_status"][idx] = status
+
+    return data
```

Comment:
This function does nothing complicated or any calculations/conditions, would just add this to the `_get_tracking_data()` function.